### ISSUE TITLE
Task: Write policy for starting a new PyLadies chapter

### ISSUE THREAD

**Issue by Mariatta** [pyladies/project-admin-logisitics#9]
# TL;DR:

The policy should answer the following questions

1. Why is the policy necessary
2. What is the purpose of a PyLadies chapter? What does it mean to start a PyLadies chapter? 
3. What are minimum responsibilities and expectation when someone requested a PyLadies chapter?
4. What kind of support can new chapter leaders expect from Global PyLadies leadership team?
5. What questions do we ask in our form? (@terezaif  is working on the form)
6. How do we review new chapter requests?
7. For accountability, how do we document new chapter creations?

Timeline: I'd ilke to have this new process in place by end of January 2020.

# Details

## 1. Why is this policy necessary?

- We had many inactive chapters, that never took off since they were created
- We find chapters with more than one organizers to be more successful, and therefore it should be part of a minimum requirement.
- We don't have clear definition of what  or why one would want to start a chapter. Generally, there is expectation for ongoing and regular community event, instead of a one time workshop.

## 2. What does it mean to start a PyLadies chapter?

Define the purpose of local PyLadies:

- help build and grow local Python community, through meetups, or sharing info through mailing lists, twitter, etc?

What are the roles of a PyLadies chapter leader?

- share news from global PyLadies leadership to their members
- organize at least one meetup per year?
- not necessarily be the one creating content/giving talk/workshop. 
- provide report once a year to global PyLadies leadership 

### 3. What are minimum responsibilities and expectation when someone requested a PyLadies chapter? 

- at least 2 co-organizers
- preferably run by women ?
- must be local (city-based). e..g Vancouver instead of BC or Canada
- namespace must be descriptive (`kuala-lumpur` instead of `kl`)?

### 4. What kind of support can new chapter leaders expect from Global PyLadies leadership team?

- as a chapter they can use PyLadies logo and name
- they will have @pyladies.com email address
- they can have pyladies subdomain
- access to organizers channel in slack
- access to organizers resources

Other privilege:
- a chapter gets one vote in PyLadies board elections

## 5. What questions do we ask in our form?

- name and email address of at least 2 organizers
- desired namespace
- intent/ability to keep the community active. e.g. if they have thought of plan idea for a meetup within the next 1 year?
- their slack.pyladies.com (confirm that the username exists. if not, they should first create an account in slack) so we can automate adding them to organizers channel.
- 

Note: @terezaif is working on the form questions

## 6. How do we review new chapter requests?

once a month? 

## 7. For accountability, how do we document new chapter creations?

Things to document:

1. who requested the new chapter
2. who reviewed/approved the new chapter request
3. did we have additional questions to organizers?
4. date we activate the chapter

**Comment by terezaif** [comment]
can you send me the questions in the old form? 

https://docs.google.com/forms/d/18GjETzcU1KeqdrOMASeyCCNyl_wvWKpunUauLt6opS8

**Comment by Mariatta** [comment]
The questions from old form are:
- Your name
- Your contact email
- How did you hear about PyLadies?
- What do you currently do with Python?
- Location of new Chapter
- Desired "namespace" for chapter - this will be for the email and subdomain setup like sf@pyladies.com and sf.pyladies.com
- Will others be helping you found and lead a PyLadies chapter? | If folks are helping you, please list the name & email of those helping you lead the chapter
- PyLadies Slack Username

**Comment by lorenanicole** [comment]
# PyLadies Request to Start Chapter Form

There's a few units of work here but I'll start with the questions for the form, edits made inline:

> The questions from old form are:
> 

Organize this into a `Organizer` section:

> * Your name

Add in:

- 'What are your pronouns (e.g. she/her, they/them)?

> * Your contact email

Add in:

- `What is your PyLadies user name? You can join at https://slackin.pyladies.com/.`

Add these questions:
- `Who are your co-organizer(s) please include their name and pronouns? We advise having at least 2 organizers a chapter to start.`
- `What are your co-organizer(s) emails?`
- `What are your co-organizer(s) PyLadies Slack user names?`

Add to a `Background` section:

> * How did you hear about PyLadies?

Change the below to:

> * What do you currently do with Python?

- `How do you and your organizer(s) use Python? Organizer(s) of all background levels are welcome! Understanding your relationship to Python is helpful for PyLadies to support your chapter as you get started.`

Add to a `Chapter Information` section:

> * Location of new Chapter

- Modify to `Please provide the city and country of the chapter. PyLadies recommends a city specific focus to prevent name collision. If you want to start a chapter with a different affiliation please specify the location and why you are choosing this.`

> * Desired "namespace" for chapter - this will be for the email and subdomain setup like [sf@pyladies.com](mailto:sf@pyladies.com) and sf.pyladies.com. 

- Modify to add helper text `The namespace should be as close to the chapter name as possible, remember in Python explicit is better than implicit!'

Add in:

- What language(s) will the chapter primarily speak?

Add in a `Programming` section with the following questions:

- When do you plan to have your first event?
- What type of event(s) do you plan to run?
- Do you have any specific support requests? For example, does your community speak a language other than English and will you want translation support? 

Add in a `Agreement` section with the following statement:

`I agree to await for official approval from PyLadies before starting to build resources online.` 

That statement is meant to try to communicate that this is an approval not a guarantee process. But I can be swayed for us to remove or soften it somehow.
